client/my-sites/themes/theme-upload/index.jsx

```diff
@@ -0,0 +1,79 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+import FilePicker from 'components/file-picker';
+import DropZone from 'components/drop-zone';
+import { localize } from 'i18n-calypso';
+import notices from 'notices';
+import debugFactory from 'debug';
+
+const debug = debugFactory( 'calypso:themes:theme-upload' );
+
+class Upload extends React.Component {
+
+	constructor( props ) {
+		super( props );
+	}
```

Comment:
Not sure you even need an explicit constructor to do this, since it's probably default behavior anyway.